airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -37,6 +37,9 @@ application {
 }
 
 dependencies {
+    compileOnly libs.lombok
```

Comment:
doesn't it make more sense to have those in the airbyte-java-connector.gradle?

Reply:
+1

Reply:
Ideally, we get rid of Lombok entirely as once we convert to Kotlin, a lot of the reason for using Lombok goes away.